src/analysis/inner_product_space/projection.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2019 Zhouhang Zhou. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Zhouhang Zhou, Frédéric Dupuis, Heather Macbeth
 -/
+import algebra.direct_sum.decomposition
```

Comment:
Consider a refactor to put the new material in new files, rather than adding imports?

Reply:
In this case I think the transitive import closure grows by just one file, the imported one.